compair/api/comparison_example.py

```diff
@@ -123,7 +130,9 @@ def get(self, course_uuid, assignment_uuid):
     def post(self, course_uuid, assignment_uuid):
         course = Course.get_active_by_uuid_or_404(course_uuid)
         assignment = Assignment.get_active_by_uuid_or_404(assignment_uuid)
-        require(CREATE, ComparisonExample(assignment=Assignment(course_id=course.id)))
+        require(CREATE, ComparisonExample(assignment=Assignment(course_id=course.id)),
+            title="Comparison Example Creation Failed",
+            message="You do not have permission to create comparison examples for this assignment since you are not the course's instructor.")
```

Comment:
"Comparison Example Not Saved" / "Your role in this course does not allow you to save comparison examples."